httpd.go

```diff
@@ -61,3 +76,9 @@ func messageHandler(p Param) (int, string) {
 		return 200, fmt.Sprintf("Message sent successfully to %s", p.Channel)
 	}
 }
+
+func sendLater(p Param, delay int64, ch chan error) (int, string) {
+	go MessageBus.Publish(NewMessage(p, ch), delay)
```

Comment:
If you agree with the proposal above, revert the change to `MessageBus.Publish()`.